libc-bottom-half/sources/chdir.c

```diff
@@ -46,15 +46,15 @@ int chdir(const char *path)
     size_t len = strlen(abs) + 1;
```

Comment:
Would would you think about renaming this to `abs_len` and removing the `+ 1`, folding the `+ 1` into the calculations below as needed? Then we could have code that does `1 + abs_len + ... + 1` which would make it a little easier to see how it relates to the `__wasilibc_cwd` comment above.

Also, there's another redundant `strlen(abs)` below which could use this.

Reply:
makes sense